turtleFSI/problems/__init__.py

```diff
@@ -187,28 +187,39 @@ def save_files_visualization(visualization_folder, dvp_, t, save_deg, mesh, **na
             tmp_t.parameters["rewrite_function_mesh"] = False
 
         if save_deg > 1:
-            # Import fenicstools
-            import warnings
-            try:
-                with warnings.catch_warnings(record=True) as w:
-                        from fenicstools import interpolate_nonmatching_mesh_any
-            except ModuleNotFoundError:
-                raise ModuleNotFoundError("For save_deg > 1 to work please install fenics tools with:\n" + \
-                                          "pip install git+https://github.com/mikaem/fenicstools")
+            print('save deg > 1 selected...')
+            print('save deg > 1 selected...')
```

Comment:
Only enough with one print-statement?

Reply:
Hey Aslak, I removed these print statements now and cleaned up the code a bit more. Should be good to go now, let me know if you have any concerns